lang/c++/api/Node.hh

```diff
@@ -135,8 +135,8 @@ class AVRO_DECL Node : private boost::noncopyable {
         doAddLeaf(newLeaf);
     }
     virtual size_t leaves() const = 0;
-    virtual const NodePtr &leafAt(int index) const = 0;
-    virtual const GenericDatum &defaultValueAt(int index) {
+    virtual const NodePtr &leafAt(size_t index) const = 0;
```

Comment:
did you consider the other change, and using int for `leaves()`?

Reply:
Yes, but this is never going to be negative and size_t is the better alternative.